### ISSUE TITLE
sails lift --prod does not link assets specified in tasks/pipeline.js

### ISSUE THREAD

**Issue by codesdk** [balderdashy/sails#3226]
My tasks/pipeline.js in my sails application looks like this

```
// CSS files to inject in order
//
// (if you're using LESS with the built-in default config, you'll want
//  to change `assets/styles/importer.less` instead.)
var cssFilesToInject = [
  'styles/**/*.css'
];


// Client-side javascript files to inject in order
// (uses Grunt-style wildcard/glob/splat expressions)
var jsFilesToInject = [

  // Load sails.io before everything else
  'js/dependencies/sails.io.js',
  'js/jquery.js',
  'js/jquery/jquery.validate.min.js',
  // Dependencies like jQuery, or Angular are brought in here
  'js/dependencies/**/*.js',


  // All of the rest of your client-side js files
  // will be injected here in no particular order.
  'js/**/*.js'
];
```

However, when I run sails lift --prod, the grunt task is not linking the css and js assets specified in the above pipeline.js file.

**Comment by metzuda** [comment]
Try: 

```javascript
// CSS files to inject in order
// (if you're using LESS with the built-in default config, you'll want
// to change assets/styles/importer.less instead.)
var cssFilesToInject = [
'/styles/*/.css'
];

// Client-side javascript files to inject in order
// (uses Grunt-style wildcard/glob/splat expressions)
var jsFilesToInject = [

// Load sails.io before everything else
'/js/dependencies/sails.io.js',
'/js/jquery.js',
'/js/jquery/jquery.validate.min.js',
// Dependencies like jQuery, or Angular are brought in here
'/js/dependencies/*/.js',

// All of the rest of your client-side js files
// will be injected here in no particular order.
'/js/*/.js'
];
```

**Comment by sgress454** [comment]
@metzuda no, those ** are there for a reason--they make sure all subdirectories are traversed.

@codesdk what exactly do you mean by "not linked"?  In production mode, Sails doesn't copy individual files to your public folder--it minifies and concatenates them into `production.min.js` and `production.min.css`.

**Comment by sailsbot** [comment]
Thanks for posting, @codesdk.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
